src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -248,19 +248,22 @@ const resolveLandingPageLinks = (
       {
```

Comment:
You probably want to remove the comment on line 245.

Reply:
Good catch